gdcdictionary/schemas/allele.yaml

```diff
@@ -57,26 +73,142 @@ properties:
     type: integer
   strand:
     $ref: _definitions.yaml#/strand
-  dbSNP_RS:
```

Comment:
Looks like dbSNP id was deprecated,  does it appear anywhere else?

Reply:
yes, https://github.com/bmeg/bmeg-dictionary/pull/9/files#diff-9e3b368863e95d3fe76dd713b2ae759fR116